### DISCUSSION TITLE
How can CMS rates for facility services be priced and does this solve the real issue?

### DISCUSSION THREAD

**Discussion by rrootdawg** [CMSgov/price-transparency-guide#692] [General]
While I understand CMS is trying to address individuals being charged facility fees for health care received outside of hospital settings as this is a real issue, posting facility fee schedules is not going to solve a "how the provider bills" issue. The issue here is provider billing behavior that borders on fraud. And, with so many variables on facility pricing and the limit to the schema how can accurate pricing be given for facility pricing based on CMS fee schedule? Can CMS produce a facility pricing fee schedule? Location, TOS, POS, diagnosis, level of service, and more all impact the final facility pricing based on CMS fee schedules. A lot of the criteria is not something a member shopping for care would have any knowledge of and is probably not even known until the service is delivered and billed. Example, what's the CMS facility pricing for a drug HCPCS code? Depends on the drug, dosage, TOS, etc. What's the CMS facility pricing for ambulance milage? Depends on the level of ambulance, zip code location, rural or urban? Current schema has no way to communicate drugs or level or service or zip code. Schema also has no way to indicate outlier pricing for DRG's. You can give a DRG rate easy enough but nothing that allows the outlier price to be given.

Due to the extreme variability in how CMS prices facility claims it's going to be almost impossible to post pricing based on CMS facility fee schedule. And posting this will more than likely cause member confusion and angst as the finial price, and thus member OOP, cannot really be determined until the service is preformed and billed.

CMS may want to reconsider their approach on addressing an issue that is really a provider billing issue. Member can still select the lower rate service at a provider office setting. But if the provider bills that as hospital based then the higher rate will get paid.

Allowing providers to bill services based in the professional office setting as facility based is the underlying issue. Members have no indication that the service being done in their professional providers office is going to be billed as hospital based until they get the bill.

Posting all the fee schedules in the world will not address this billing issue.

**Comment by AshleyOboe** [comment]
Hi @rrootdawg - a couple of things that might be helpful:
* The rates that are posted in the MRF files are for commercial payers in the commercial market, where rates are not set or regulated by CMS.  
* We've observed also that the facility payments can be more complicated than the MRF schema allows.  The general sense from hospital executives is that for complex procedures like spine & joint surgery, there are other factors like patient acuity and carve-outs are written into provider contracts that impact the pricing.  The base institutional prices may not reflect those, however, comparing base prices can give a directional sense of cost. 
* In my experience, if you have a procedure at a hospital the facility will first contact you asking for payment or giving you an estimate.  This isn't mandated but I don't think it's super common to see a physician for an office visit and get a facility bill.  What's more common is that you see a physician for an office visit who has been employed by a large health system, and the rate is higher than an independent physician because the health system had more negotiating power in the market.  The data can be very helpful for this use case, because professional office visit rates can be compared apples to apples.  We can also see which physicians may be associated with multiple health systems or reported under more than one tax entity.  
* I haven't looked into ambulance but am curious how they reported those rates. I think technically the MRF schema could be sufficient to report if they just added information about whether it was a base rate or per-mile rate but I'm guessing they didn't do that.  Is there a code you're looking at?

**Reply by rrootdawg** [reply]
AshleyOboe,

Commercial payers will use CMS fee schedule rates in certain contract situations. And specifically for facility often times you can't get a price on a service until you have a claim to get all the other criteria to then input into a pricer to get a price for the service back.

My example of ambulance is just one example. The ambulance milage code will be priced differently depending on type of ambulance code, the service being provided, location, urban or rural, etc. MRF has no way to call out pricing differentials based on zip codes or even type of other service code being submitted. It's not really a bundling situation. It's basing one price off of another service code. Again, information that would become known once the claim was submitted and probably not information a member would even be aware of or even how to factor it into their shopping.

To a member it's simple a provider, a service code, and you get a price. All the other stuff that impacts pricing is really not something members are aware of and a lot of what impacts pricing isn't know until the service is actually rendered by the provider.
